client/lib/translator-jumpstart/index.js

```diff
@@ -91,6 +91,11 @@ const communityTranslatorJumpstart = {
 			return displayedTranslationFromPage;
 		}
 
+		if ( 'boolean' === typeof optionsFromPage.textOnly && optionsFromPage.textOnly ) {
+			debug( 'respecting textOnly for string "' + originalFromPage + '"' );
+			return displayedTranslationFromPage;
+		}
+
```

Comment:
Nice. 👍  For #21591 I'll need this to disable the translator for the translator ;)

Reply:
Really? I actually only used this to identify textOnly entries (using `localStorage.setItem('debug', 'calypso:community-translator')`) that I'd like to eliminate eventually. Having text wrapped in `<Translatable>` should still keep it usable.

Reply:
Ah, I see. I think there should be some way to circumvent wrapping a translation. I can only think of the CT dialog box as an example. Kinda like Translatable inception in that case :)

This seems too elaborate...

```
const closeText = this.props.translate( 'Close' );
<Button primary onClick={ this.closeDialog }>
   { closeText.toString() }
</Button>
```